.github/workflows/cd.yml

```diff
@@ -0,0 +1,19 @@
+name: CD
+
+on:
+  push:
+    tags:
+      - '**'
+
+jobs:
+
+  check-tag-version-job:
+    name: Check Release Tag
+    uses: ./.github/workflows/check-release-tag.yml
+
+  cd-job:
+    name: Continuous Delivery
+    needs: [ ci-job ]
```

Comment:
```suggestion
```
This needs can't be satisfied, this is a bug in the latest version of the `python-toolbox`.
Note: bug have been fixed but new version still needs to be released